distributed/config.yaml

```diff
@@ -17,6 +17,20 @@ multiprocessing-method: forkserver
 # Communication options
 tcp-timeout: 30         # seconds delay before calling an unresponsive connection dead
 default-scheme: tcp
+require-encryption: False   # whether to require encryption on non-local comms
```

Comment:
@jbednar, @stuartarchibald, these are the configuration file entries that I'm proposing to add. Note `ciphers` is an arbitrary cipher string in OpenSSL format.